exporter/debugexporter/debug_exporter.go

```diff
@@ -0,0 +1,83 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package debugexporter // import "go.opentelemetry.io/collector/exporter/debugexporter"
+
+import (
+	"context"
+
+	"go.uber.org/zap"
+
+	"go.opentelemetry.io/collector/config/configtelemetry"
+	"go.opentelemetry.io/collector/exporter/internal/otlptext"
+	"go.opentelemetry.io/collector/pdata/plog"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+)
+
+type debugExporter struct {
```

Comment:
Why can't we move this one to internal?

Reply:
We could, it just didn't seem to really fit, i guess i could create an `exporter/internal/loggingcommonexporter` package or something like it

Reply:
Yeah, something like that. I'm just thinking of how we can reduce the code duplication. Looks like it's technically possible. If so, I'd merge this PR and do that later rather than https://github.com/open-telemetry/opentelemetry-collector/pull/8375